contracts/libraries/LiquidityDeltaAccounting.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {BalanceDelta} from "@uniswap/v4-core/src/types/BalanceDelta.sol";
+
+import "forge-std/console2.sol";
+
+library LiquidityDeltaAccounting {
+    function split(BalanceDelta liquidityDelta, BalanceDelta callerFeesAccrued, BalanceDelta totalFeesAccrued)
```

Comment:
i like this library a lot and how its used. its a super minor nit but i think we should call it `discern` and switch the parameters so the syntax becomes:

`liquidityDelta.discern(totalFeesAccrued, callerFeesAccrued)`